aggregator/src/test/java/com/redhat/cloud/notifications/db/EmailAggregationRepositoryTest.java

```diff
@@ -87,29 +93,10 @@ void testApplicationsWithPendingAggregationAccordingOrgPref(boolean useSystemEnd
 
     @ParameterizedTest
     @ValueSource(booleans = {true, false})
-    void testApplicationsWithPendingAggregationUsingDefaultSystemIntegration(boolean useSystemEndpoint) {
-        configureTimePref(dailyEmailAggregationJob.computeScheduleExecutionTime());
-        Event event1 = resourceHelpers.addEventEmailAggregation(ORG_ID, BUNDLE_NAME, APP_NAME, dailyEmailAggregationJob.computeScheduleExecutionTime().minusMinutes(5), PAYLOAD1.toString(), useSystemEndpoint);
-        Event event2 = resourceHelpers.addEventEmailAggregation(ORG_ID, BUNDLE_NAME, APP_NAME, dailyEmailAggregationJob.computeScheduleExecutionTime().minusMinutes(5), PAYLOAD2.toString(), useSystemEndpoint);
-        resourceHelpers.addEventEmailAggregation("other-org-id", BUNDLE_NAME, APP_NAME, dailyEmailAggregationJob.computeScheduleExecutionTime().minusMinutes(5), PAYLOAD2.toString(), useSystemEndpoint);
-        resourceHelpers.addEventEmailAggregation(ORG_ID, "other-bundle", APP_NAME, dailyEmailAggregationJob.computeScheduleExecutionTime().minusMinutes(5), PAYLOAD2.toString(), useSystemEndpoint);
-        resourceHelpers.addEventEmailAggregation(ORG_ID, BUNDLE_NAME, "other-app", dailyEmailAggregationJob.computeScheduleExecutionTime().minusMinutes(5), PAYLOAD2.toString(), useSystemEndpoint);
-
-        List<AggregationCommand> keys = emailAggregationResources.getApplicationsWithPendingAggregationAccordingOrgPref(dailyEmailAggregationJob.computeScheduleExecutionTime());
-        assertEquals(4, keys.size());
-        Application application = resourceHelpers.findApp(BUNDLE_NAME, APP_NAME);
-
-        List<AggregationCommand> matchedKeys = keys.stream().filter(k -> ORG_ID.equals(k.getOrgId())).filter(k -> (((EventAggregationCriterion) k.getAggregationKey()).getApplicationId().equals(application.getId()))).collect(Collectors.toList());
-        assertEquals(1, matchedKeys.size());
-        assertEquals(BUNDLE_NAME,  matchedKeys.get(0).getAggregationKey().getBundle());
-        assertEquals(APP_NAME, matchedKeys.get(0).getAggregationKey().getApplication());
-
-        resourceHelpers.deleteEvent(event1);
-        resourceHelpers.deleteEvent(event2);
-
-        keys = emailAggregationResources.getApplicationsWithPendingAggregationAccordingOrgPref(dailyEmailAggregationJob.computeScheduleExecutionTime());
-        assertEquals(3, keys.size());
-        matchedKeys = keys.stream().filter(k -> ORG_ID.equals(k.getOrgId())).filter(k -> (((EventAggregationCriterion) k.getAggregationKey()).getApplicationId().equals(application.getId()))).collect(Collectors.toList());
-        assertEquals(0, matchedKeys.size());
+    void testAggregationWithSubscribersOnly(boolean useSystemEndpoint) {
+        when(aggregatorConfig.isFetchAggregationsWithAtLeastOneSubscriber()).thenReturn(useSystemEndpoint);
```

Comment:
```suggestion
    void testAggregationWithSubscribersOnly(boolean atLeastOneSubscriber) {
        when(aggregatorConfig.isFetchAggregationsWithAtLeastOneSubscriber()).thenReturn(atLeastOneSubscriber);
```
Rename to clarify the boolean's purpose